js/app/demo/app.jsx

```diff
@@ -110,12 +113,15 @@ define(['react', 'jsx!editor', 'jsx!messages', 'jsx!fixedCode', 'jsx!configurati
                 });
             }.bind(this))
         },
+        componentDidUpdate: function() {
+            window.localStorage.setItem('linterDemoState', JSON.stringify(this.state));
+        },
         render: function() {
             return (
                 <div className="container editorRow">
                     <div className="row">
                         <div className="col-md-7">
-                            <Editor onChange={this.handleChange} text="var foo = bar;" errors={this.state.messages} />
+                            <Editor onChange={this.handleChange} text={this.initialText} errors={this.state.messages} />
```

Comment:
I'm not sure exactly what's going on, but when I set this to `text={this.state.text}`, the editor breaks and stops being editable after entering a single character. However, it is editable if the prop is static.